tests/kernel/arm_zero_latency_irqs/src/arm_zero_latency_irqs.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <arch/cpu.h>
+#include <arch/arm/cortex_m/cmsis.h>
+
+static volatile int test_flag;
+
+void arm_zero_latency_isr_handler(void *args)
+{
+	ARG_UNUSED(args);
+
+	test_flag = 1;
+}
+
+void test_arm_zero_latency_irqs(void)
+{
+	/* Determine an NVIC IRQ line that is not currently in use. */
+	int i, key;
+
+	zassert_false(test_flag, "Test flag not initialized to zero\n");
+
+	for (i = CONFIG_NUM_IRQS - 1; i >= 0; i--) {
+		if (NVIC_GetEnableIRQ(i) == 0) {
```

Comment:
Can you add a comment as to why `GetEnableIRQ()` allows you to find an available interrupt?

Reply:
Something like what we have in line 22?

Reply:
Or a more detailed comment, e.g.
 `/*
   *Interrupts configured statically with IRQ_CONNEC(T..) are automatically enabled. NVIC_GetEnableIRQ() returning
   *false implies that the IRQ line is not currently in use by Zephyr.
  */`